src/services/safe_border.py

```diff
@@ -111,14 +111,19 @@ def _get_earliest_slashed_epoch_among_incomplete_slashings(self) -> Optional[Epo
 
         return earliest_predicted_epoch
 
-    # If there are no so many validators in exit queue we can be quite sure that
-    # slashing has started not earlier than 8,192 epochs or ~36 days ago
+    # The exit period for a specific validator may be equal to the MIN_VALIDATOR_WITHDRAWAL_DELAY.
+    # This means that there are so many validators in the queue that the exit epoch moves with the withdrawable epoch,
+    # and we cannot detect when slashing has started.
     def _predict_earliest_slashed_epoch(self, validator: Validator) -> Optional[EpochNumber]:
         exit_epoch = int(validator.validator.exit_epoch)
         withdrawable_epoch = int(validator.validator.withdrawable_epoch)
 
         exited_period = withdrawable_epoch - exit_epoch
-        is_slashed_epoch_undetectable = exited_period > MIN_VALIDATOR_WITHDRAWABILITY_DELAY
+
+        if exited_period < MIN_VALIDATOR_WITHDRAWABILITY_DELAY:
+            raise WrongExitPeriod("exit_epoch and withdrawable_epoch are too close")
+
+        is_slashed_epoch_undetectable = exited_period == MIN_VALIDATOR_WITHDRAWABILITY_DELAY
```

Comment:
```suggestion
        is_slashed_epoch_undetectable = exited_period != MIN_VALIDATOR_WITHDRAWABILITY_DELAY
```

Reply:
If I understand correctly tests should fail here, but they passed.
Add to tests this case

Reply:
It should be equal so the condition is correct
see [discussion](https://discord.com/channels/761182643269795850/1072446086423257088/1082294765493694514)